### DISCUSSION TITLE
Move Collections into Eloquent/Database

### DISCUSSION THREAD

**Discussion by duncanmcclean** [statamic-rad-pack/runway#192] [Feature Requests]
One of my ideas for Runway is to build some kind of migration tool where you can select a collection & Runway will handle the process of moving it into Eloquent & setting it up with Runway.

Things like:

* Bootstrapping the Eloquent model & migrations (based on the collection blueprint)
* Copy over the blueprint file for Runway
* Configuring Runway to use the same settings as the collection
    * Example: if you're using front-end routing, we'd keep the route the same
* Migrating the entries themselves into the database
* **Depending on the complexity:** change any Antlers code that calls `{{ collection:whatever }}` to `{{ runway:whatever }}`

I think this may be helpful for folks who have built their sites with entries but have later realised they need to swap out to a database as they scale or need greater control over how they control / deal with their 'entries'.

I'm unsure how useful this would actually be but it's just a thought I've had. If you'd like to see it, give this discussion a  👍  and feel free to leave any comments you have down below.

**Comment by jolora** [comment]
I'd never considered this but it would be something I'd definitely have used in the past if it had been possible

**Comment by duncanmcclean** [comment]
This feature will be available in Runway 8, which'll be released shortly.
